senders/mattermost/client.go

```diff
@@ -0,0 +1,9 @@
+package mattermost
+
+import "github.com/mattermost/mattermost-server/v6/model"
+
+// Client is wrapper over model.Client4.
+type Client interface {
```

Comment:
lose interface naming :(

Reply:
why single file?

Reply:
>lose interface naming :(

Read it with package name - `mattermost.Client`, sounds good

Reply:
> why single file?

I think it is ok to separate types by files, because mattermost.go is about Sender impl. Maybe rename mattermost.go to sender.go 🤔 Think it would be better

Reply:
interfaces in go ended on "er"

Reply:
Renamed file, I think it is better https://github.com/moira-alert/moira/pull/791/commits/8f199548289cfcfe0addc17d9886c1458a66e7bc

Reply:
- Why separate file?

[and about naming](https://go.dev/doc/effective_go) Interface names